docs/src/development/ssh/ssh-keys.md

```diff
@@ -70,7 +70,7 @@ generate a key and have it added to your Platform.sh account automatically.
 1. Run the following commands (replacing `PATH_TO_YOUR_KEY` with the location you copied):
```

Comment:
:nail_care:
The format of the variable is not following our guidelines, we might as well fix that.
```suggestion
1. Run the following commands (replacing `<PATH_TO_YOUR_KEY>` with the location you copied):
```